src/main/java/org/craftercms/studio/impl/v1/service/content/ContentServiceImpl.java

```diff
@@ -917,10 +912,10 @@ protected String copyContent(String site, String fromPath, String toPath, Set<St
             }
 
             applicationContext.publishEvent(new ContentEvent(securityService.getAuthentication(), site, toPath));
-        } catch(ServiceLayerException | UserNotFoundException eServiceLayerException) {
-            logger.info("General Error while copying content for site {0} from {1} to {2}, new name is {3}",
-                eServiceLayerException, site, fromPath, toPath, copyPath);
-            throw eServiceLayerException;
+        } catch(ServiceLayerException | UserNotFoundException e) {
```

Comment:
Should UserNotFoundException extend ServiceLayerException?

Reply:
I'm not sure. Let's take that as an action item to investigate and adjust as needed.